cpu/stm32_common/periph/flash_common.c

```diff
@@ -56,3 +57,16 @@ void _lock(void)
         CNTRL_REG |= CNTRL_REG_LOCK;
     }
 }
+
+void _wait_for_pending_operations(void)
+{
+    if ((FLASH->SR & FLASH_SR_BSY) == FLASH_SR_BSY) {
```

Comment:
Maybe just:
```suggestion
    if (FLASH->SR & FLASH_SR_BSY) {
```
?

Reply:
Yep, makes sense with how its called afterwards to. Changed!